server/tailsql/options.go

```diff
@@ -610,3 +627,42 @@ func DefaultCheckQuery(q Query) (Query, error) {
 	}
 	return q, nil
 }
+
+// Queryable is the interface used to issue SQL queries to a database.
+type Queryable interface {
+	// Query issues the specified SQL query in a transaction and returns the
+	// matching result set, if any.
+	Query(ctx context.Context, sql string) (RowSet, error)
```

Comment:
i guess it doesn't take params, but could?

Reply:
Good call, done.